win32/libssh2.dsp

```diff
@@ -0,0 +1,308 @@
+# Microsoft Developer Studio Project File - Name="libssh2" - Package Owner=<4>

```

Comment:
I'm curious we need this in the repo (and I suppose the entire win32 directory at this point).  Does cmake generate everything we need?